django/applications/catmaid/control/importer.py

```diff
@@ -1260,7 +1261,7 @@ def import_projects( user, pre_projects, tags, permissions,
                               mirror.save()
                       else:
                           raise ValueError("Invalid action for known mirror: " +
-                              known_stack_action)
+                             str(known_stack_action))
```

Comment:
Was bytes or None rather than string. Seemed an obvious bugfix.

Reply:
Yes, indeed, this is done also a few lines above for another exception. It being `bytes` is surprising though, but unrelated to this fix.